DataCapturing/Movebis/MovebisServerConnection.swift

```diff
@@ -143,9 +143,13 @@ public class MovebisServerConnection: ServerConnection {
         loadMeasurementGroup.enter()
         persistenceLayer.load(measurementIdentifiedBy: measurement.identifier) { measurement in
             debugPrint("loaded measurement \(measurement.identifier)")
-            let payload = self.serializer.serializeCompressed(measurement)
-            request.append(payload, withName: "fileToUpload", fileName: "\(self.installationIdentifier)_\(measurement.identifier).cyf", mimeType: "application/octet-stream")
-            loadMeasurementGroup.leave()
+            do {
+                let payload = try self.serializer.serializeCompressed(measurement)
+                request.append(payload, withName: "fileToUpload", fileName: "\(self.installationIdentifier)_\(measurement.identifier).cyf", mimeType: "application/octet-stream")
+                loadMeasurementGroup.leave()
+            } catch {
+                fatalError("Unable to serialize measurement \(measurement.identifier). Error \(error).")
+            }
```

Comment:
Just added an exception handler here. In the new version I intend to work more with clean exceptions, rather then the fatalError method I have used so far. I did not know how this worked exactly when I started the project.